ion-schema/src/main/kotlin/com/amazon/ionschema/internal/ConstraintFactory.kt

```diff
@@ -25,7 +25,7 @@ internal interface ConstraintFactory {
     /**
      * If [name] is a recognized constraint name, returns `true`, otherwise `false`.
```

Comment:
```suggestion
     * If [name] is a recognized constraint name, returns `true`, otherwise `false` based on the ISL version that given [schema] uses.
```